src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -20,6 +24,9 @@ export class DotEditLayoutComponent implements OnInit {
     @ViewChild('templateName') templateName: ElementRef;
 
     form: FormGroup;
+    initialFormValue: FormGroup;
+    disableUpdateButton = true;
```

Comment:
Let's rename this property to ```canSave``` and let's use the same across the app... cc: @freddyucv @luisrojascr